addons/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/status/S3BinaryManagerStatusProbe.java

```diff
@@ -38,24 +38,15 @@
     @Override
     public ProbeStatus run() {
         Map<String, BlobProvider> providers = Framework.getService(BlobManager.class).getBlobProviders();
-        boolean someSuccess = false;
         for (Map.Entry<String, BlobProvider> en : providers.entrySet()) {
             String id = en.getKey();
             BlobProvider blobProvider = en.getValue();
             BinaryManager bm = blobProvider.getBinaryManager();
-            if (bm == null || !(bm instanceof S3BinaryManager)) {
-                continue;
-            }
-            if (!((S3BinaryManager) bm).canAccessBucket()) {
+            if (bm instanceof S3BinaryManager && !((S3BinaryManager) bm).canAccessBucket()) {
                 return ProbeStatus.newFailure("S3BinaryManager cannot access the configured bucket: " + id);
             }
-            someSuccess = true;
-        }
-        if (someSuccess) {
-            return ProbeStatus.newSuccess("S3BinaryManager can access the configured buckets");
-        } else {
-            return ProbeStatus.newFailure("No S3BinaryManager bucket configured");
         }
+        return ProbeStatus.newSuccess("S3BinaryManager can access the configured buckets");
```

Comment:
Isn't it weird to have "S3BinaryManager can access the configured buckets" as status message if there is no S3BinaryManager at all?

Reply:
Yes it's a bit weird but I didn't know if anything could be parsing these messages. But indeed given that we're changing the behavior if there's no binary manager we might as well cleanup the message. I'll do that.

Reply:
@troger now updated